website/docs/r/wafregional_web_acl.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: "aws"
+page_title: "AWS: aws_wafregional_web_acl"
+sidebar_current: "docs-aws-resource-wafregional-web-acl"
+description: |-
+  Provides a AWS WAF Regional web access control group (ACL) resource for use with ALB.
+---
+
+# aws\_wafregional\_web\_acl
+
+Provides a WAF Regional Web ACL Resource for use with Application Load Balancer.
+
+## Example Usage
+
+```hcl
+resource "aws_wafregional_ipset" "ipset" {
+  name = "tfIPSet"
+  
+  ip_set_descriptors {
+    type  = "IPV4"
+    value = "192.0.7.0/24"
+  }
+}
+
+resource "aws_wafregional_rule" "wafrule" {
+  depends_on  = ["aws_wafregional_ipset.ipset"]
+  name        = "tfWAFRule"
+  metric_name = "tfWAFRule"
+  
+  predicates {
+    data_id = "${aws_wafregional_ipset.ipset.id}"
+    negated = false
+    type    = "IPMatch"
+  }
+}
+
+resource "aws_wafregional_web_acl" "wafacl" {
+  depends_on  = ["aws_wafregional_ipset.ipset", "aws_wafregional_rule.wafrule"]
```

Comment:
This is redundant because the relationship is already built by the reference `${aws_wafregional_rule.wafrule.id}` below and further reference inside the rule. Do you mind removing it?